zerver/views/messages.py

```diff
@@ -519,7 +523,7 @@ def convert_term(elem: Union[Dict[str, Any], List[str]]) -> Dict[str, Any]:
             # that supports user IDs. Relevant code is located in static/js/message_fetch.js
             # in handle_user_ids_supported_operators function where you will need to update
             # user_id_supported_operator, or user_ids_supported_operator array.
-            user_id_supported_operator = ['sender', 'group-pm-with']
+            user_id_supported_operator = ['sender', 'group-pm-with', 'stream']
```

Comment:
We should do a prep commit changing this from `user_id_supported_operator` to `operators_supporting_ids`

Reply:
Done. That variable name is great!